routes/mobileapplite1.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * mobileapp/lite provides page content for a potential lite Mobile App.
+ * The goal is to avoid having to use a web view and style the content natively inside the app
+ * using plain TextViews.
+ * The payload should not have any extra data, and should be easy to consume by the Lite App.
+ *
+ * Status: Prototype -- not ready for production
+ * Currently using the mobileview action MW API, and removing some data we don't display.
+ * TODO: Try Parsoid
+ * TODO: Split the "text" objects of each section into paragraph and table objects
+ * TODO: add some transformations that currently are being done by the apps and remove some more unneeded data
+ */
+
+
+'use strict';
+
+
+var BBPromise = require('bluebird');
+var preq = require('preq');
+var domino = require('domino');
+var sUtil = require('../lib/util');
+var util = require('util');
+
+// shortcut
+var HTTPError = sUtil.HTTPError;
+
+
+/**
+ * The main router object
+ */
+var router = sUtil.router();
+
+/**
+ * The main application object reported when this module is require()d
+ */
+var app;
+
+
+/**
+ * A helper function that obtains the HTML from the MW API and
+ * loads it into a domino DOM document instance.
+ *
+ * @param {String} domain the domain to contact
+ * @param {Object} params an Object with all the query parameters for the MW API
+ * @return {Promise} a promise resolving as the HTML element object
+ */
+function apiGet(domain, params) {
+
+    // get the page from the MW API
+    return preq.get({
+        uri: 'http://' + domain + '/w/api.php',
+        query: params
+    }).then(function(callRes) {
```

Comment:
No need for this `then()` block, you can safely return what `preq.get()` returns and use the `callRes` object there. A `then()` block would be useful here only if you plan to somehow process the result before returning it.